src/krati_Shukla/Number.java

```diff
@@ -0,0 +1,48 @@
+package krati_Shukla;
+
+public class Number {
+	
+	void displayEven(int no) {
+		System.out.println("Even numbers are : ");
+		for (int i=no; i<=15; i=i+2) {
```

Comment:
it should be
for(int number=1;number<=endNumber; number++){
   if(number%2 == 0)
}